### DISCUSSION TITLE
Fields Not Updating

### DISCUSSION THREAD

**Discussion by ismaeltj9** [filamentphp/filament#11885] [Help] [answered]
### Package

Form builder

### Package Version

3.2

### How can we help you?

Hello, I have a resource with hasmany and repeater to show this relations, when I change a value of some itens, a field is updated and showed instantly in the form, but when I change a value of main form it is not.
I dont know why, I appreciate any help.
![g](https://github.com/filamentphp/filament/assets/24549114/a11d90a2-a770-41c4-9120-9617df3c00d2)

**Comment by ismaeltj9** [comment, marked as answer]
I used follow code in edit page and for me it's ok, maybe not the best solution.

    protected function getRedirectUrl(): string
    {
        return $this->getResource()::getUrl('edit', ['record' => $this->getRecord()]);
    }

**Comment by tahacankurt** [comment]
I am experiencing the same problem. i have to update some fields on boot method of model class but unfortunately those field updates are not appearing on front end form
